src/auxiliary/math.jl

```diff
@@ -5,6 +5,93 @@
 @muladd begin
 #! format: noindent
 
+"""
+    sqrt_(x::Real)
```

Comment:
I think I've written this somewhere but I would prefer not having to use an additional `sqrt_` and `log_` but just shadow them in Trixi.jl.